tests/functional/complex_list_comp.py

```diff
@@ -0,0 +1,38 @@
+# pylint: disable=undefined-variable, missing-docstring
+
+
+def fcn_to_fail():
+    result = [(x, y) for x in range(10) for y in range(5) if x * y > 10]  # [complex-list-comp]
+    return result
+
+
+def one_to_pass():  # should be fine
```

Comment:
These `should be fine` comments seem superfluous since (1) we mark the failures and (2) the function name is "one_to_pass"